src/base/ops.rs

```diff
@@ -374,6 +374,46 @@ componentwise_binop_impl!(Sub, sub, ClosedSub;
                           SubAssign, sub_assign, sub_assign_statically_unchecked, sub_assign_statically_unchecked_mut;
                           sub_to, sub_to_statically_unchecked);
 
+/// Matrices and vectors implement the `iter::Sum` trait. 
+/// 
+/// # Static matrices
+///
+/// ```.rust
```

Comment:
You can drop the `.rust`. Unless a different language is explicitly specified, all code comments default to being interpreted as rust code.